src/main/scala/rules/Brancher.scala

```diff
@@ -1,190 +1,189 @@
-            v1.decider.setCurrentBranchCondition(negatedCondition)
-
-            fElse(stateConsolidator.consolidateIfRetrying(s1, v1), v1)
-          })
-        }
-      } else {
-        _ => Unreachable()
-      }
-
-    val elseBranchFuture: Future[Seq[VerificationResult]] =
-      if (executeElseBranch) {
-        if (parallelizeElseBranch) {
-          /* [BRANCH-PARALLELISATION] */
-          throw new RuntimeException("Branch parallelisation is expected to be deactivated for now")
-//          v.verificationPoolManager.queueVerificationTask(v0 => {
-//            v0.verificationPoolManager.runningVerificationTasks.put(elseBranchVerificationTask, true)
-//            val res = elseBranchVerificationTask(v0)
-//
-//            v0.verificationPoolManager.runningVerificationTasks.remove(elseBranchVerificationTask)
-//
-//            Seq(res)
-//          })
-        } else {
-          new SynchronousFuture(Seq(elseBranchVerificationTask(v)))
-        }
-      } else {
-        CompletableFuture.completedFuture(Seq(Unreachable()))
-      }
-
-    (if (executeThenBranch) {
-      executionFlowController.locally(s, v)((s1, v1) => {
-        v1.decider.prover.comment(s"[then-branch: $cnt | $condition]")
-        v1.decider.setCurrentBranchCondition(condition)
-
-        fThen(stateConsolidator.consolidateIfRetrying(s1, v1), v1)
-      })
-    } else {
-      Unreachable()
-    }) && {
-
-      /* [BRANCH-PARALLELISATION] */
-      if (parallelizeElseBranch) {
-//          && v.verificationPoolManager.slaveVerifierPool.getNumIdle == 0
-//          && !v.verificationPoolManager.runningVerificationTasks.containsKey(elseBranchVerificationTask)
-//                /* TODO: This attempt to ensure that the elseBranchVerificationTask is not already
-//                 *       being executed is most likely not thread-safe since checking if a task
-//                 *       is still in the queue and canceling it, if so, is not an atomic operation.
-//                 *       I.e. the task may be taken out of the queue in between.
-//                 */
-
-        throw new RuntimeException("Branch parallelisation is expected to be deactivated for now")
-
-//        /* Cancelling the task should result in the underlying task being removed from the task
-//         * queue/executor
-//         */
-//        elseBranchFuture.cancel(true)
-//
-//        /* Run the task on the current thread */
-//        elseBranchVerificationTask(v)
-      } else {
-        var rs: Seq[VerificationResult] = null
-        try {
-          rs = elseBranchFuture.get()
-        } catch {
-          case ex: ExecutionException =>
-            ex.getCause.printStackTrace()
-            throw ex
-        }
-
-        assert(rs.length == 1, s"Expected a single verification result but found ${rs.length}")
-        rs.head
-      }
-    }
-  }
-}
+// This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Try to minimise the diffs, and e.g. revert any whitespace changes. Did anything in `Brancher.scala` actually change?

Reply:
This might be because I use `LF` as the line separator and the file was using `CRLF` before, I'll change it so hopefully it doesn't show the whole file as changed!